src/material/stepper/stepper.md

```diff
@@ -222,3 +222,19 @@ is given `role="tab"`, and the content that can be expanded upon selection is gi
 step content is automatically set based on step selection change.
 
 The stepper and each step should be given a meaningful label via `aria-label` or `aria-labelledby`.
+
+When a stepper is used on a smaller screen, verical steppers are preferred over
```

Comment:
verical -> vertical

Reply:
```markdown
Prefer vertical steppers when building for small screen sizes, as horizontal steppers typically take up significantly more horizontal space, thus introducing horizontal scrolling. Applications with multiple scrolling dimensions make content harder to consume for some users. See the [Responsive Stepper section](#responsive-stepper) above for an example on building a stepper that adjusts its layout based on viewport size.
```
* Eliminates passive voice (e.g. "stepper is used", "steppers are preferred")
* Adds some context on _why_ folks would want to be careful with horizontal steppers on small screens
* Links to the example above